### ISSUE TITLE
Arduino AVR Boards post 1.6.11 bloats build - consumes memory

### ISSUE THREAD

**Issue by Larry-M** [arduino/Arduino#5392]
Reference https://discuss.inventables.com/t/inventables-fork-of-grbl-too-large-32-266-bytes-when-compiled/28148/4

**Comment by Chris--A** [comment]
Yes I have noticed this too. Some code that was really efficient in GCC 4.3.x is now bloating out and adds an extra 5kb to the code and slows it down considerably. Over the weekend I'll go hunting for the cause. I wonder if the LTO is causing issues.
